examples/htool_security_certificates.h

```diff
@@ -0,0 +1,115 @@
+#ifndef LIBHOTH_EXAMPLES_HTOOL_SECURITY_CERTIFICATES_H_
+#define LIBHOTH_EXAMPLES_HTOOL_SECURITY_CERTIFICATES_H_
+
+#include <stdint.h>
+
+#include "htool_security_v2_defs.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// Forward declaration
+struct htool_invocation;
+struct security_v2_param;
+
+#define KEY_INFO_SIZE 4
+
+#define ALIAS_KEY_V0_SIZE 192
+
+#define ALIAS_KEY_V1_SIZE 224
+
+#define MAX_ALIAS_KEY_SIZE ALIAS_KEY_V1_SIZE
+
+struct key_cert_signature_header {
```

Comment:
should we just return sized buffers from the commands rather than parse them into structs defined here? Goal is to keep the tool simple and compact.